SpecCheck.py

```diff
@@ -763,6 +768,14 @@ def check_spec(self, pkg, spec_file, spec_lines=None):
 '''Make check or other automated regression test should be run in %check, as
 they can be disabled with a rpm macro for short circuiting purposes.''',
 
+'deprecated-use-of-%run_ldconfig',
+'''According to the packaging conventions, the use of %run_ldconfig
+is deprecated. Please use /sbin/ldconfig instead, or
+
+%post(un) -p /sbin/ldconfig
+
+in the case where ldconfig is the only command to be executed.''',
```

Comment:
This has actually been changed in Fedora: https://fedoraproject.org/wiki/Changes/Removing_ldconfig_scriptlets

Please remove this change from this PR.

Reply:
Are those scriptlets in default rpm so it could be recommended globaly?

Reply:
Not yet, no. It's being looked into, though.